### PR TITLE
radar.io -> radar.com

### PR DESCRIPTION
**Pull request by kochis**
Update domains to `.com`

### CODE REVIEW

File: .github/workflows/documentation.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
           USE_SSH: true
           GIT_USER: git
         run: |
-          git config --global user.email "andrew@radar.io"
+          git config --global user.email "andrew@radar.com"
```
Comment (kochis) [inline .github/workflows/documentation.yml:44]: I'll probably leave this one as `.io` for now

File: docs/api.mdx [hunk 2]
```diff
@@ -411,7 +411,7 @@ Publishable
 ###### Sample request
 
 ```bash
-curl "https://api.radar.io/v1/geocode/forward?query=20+jay+st+brooklyn+ny" \
+curl "https://api.radar.com/v1/geocode/forward?query=20+jay+st+brooklyn+ny" \
```
Comment (kochis) [inline docs/api.mdx:414]: I'll revert the api changes for now

File: docs/integrations/olo.mdx [hunk 3]
```diff
@@ -8,9 +8,9 @@ Radar can [start and stop trips](/trip-tracking) for customer arrival tracking w
 
 Configure a Olo webhook to deliver to https://api.radar.io/webhooks/olo for the relevant Order Events via the Development Tools section of the Olo Dashboard (requires 2 factor authentication and at least Brand Manager permission). When order events are triggered in Olo, Olo will call the Radar webhook. Collect your brand ID from Olo.
```
Comment (jsani-radar) [inline docs/integrations/olo.mdx:9]: nice for omitting this 👍🏽

**Review by jsani-radar — APPROVED** [review]
🙏🏽

feel free to merge

**Pull request merged**